smithy-linters/src/test/resources/software/amazon/smithy/linters/errorfiles/input-output-structure-reuse.errors

```diff
@@ -1,2 +1,2 @@
-[WARNING] smithy.example#GetFooInput: This structure is the input of `smithy.example#GetFoo`, but it is not marked with the @input trait. The @input trait gives operations more flexibility to evolve their top-level input members in ways that would otherwise be backward incompatible. | InputOutputStructureReuse
-[WARNING] smithy.example#GetFooOutput: This structure is the output of `smithy.example#GetFoo`, but it is not marked with the @output trait. | InputOutputStructureReuse
+[WARNING] smithy.example#GetFooInput: This structure is the input of `smithy.example#GetFoo`, but it is not marked with the @input trait. The @input trait gives operations more flexibility to evolve their top-level input members in ways that would otherwise be backward incompatible. | InputOutputStructureReuse.INPUT.smithy.example#GetFoo
+[WARNING] smithy.example#GetFooOutput: This structure is the output of `smithy.example#GetFoo`, but it is not marked with the @output trait. | InputOutputStructureReuse.OUTPUT.smithy.example#GetFoo
```

Comment:
I don't see how the shape ID of the operation it's used by is that helpful here. The intent of the validator is that shapes used as input or output should have the corresponding input/output trait, so suppressing specific combinations of these won't help.

Can we also do `InputOutputStructureReuse.Output` and `InputOutputStructureReuse.Input` instead of using all caps?

Reply:
It's necessary for uniqueness. The same structure can be the input or output for multiple operations (bad design but too late to fix for many shipped services), and would raise a new violation for each of those. Also we don't want to aggregate that case or any similar ones, because then it would hide any future additions to that aggregation

I'll fix the casing after syncing with you about it throughout

Reply:
Hm ok. It might be good to use just the name of the operation to help mitigate cases where a team changes the namespace for external use vs internal use (e.g., `InputOutputStructureReuse.Output.GetFoo`)

Reply:
Ok sure, I'll make this change on next commit 👍